ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge-no-default-policy.cfg

```diff
@@ -190,9 +186,10 @@ ip prefix-list PL-LOOPBACKS-EVPN-OVERLAY
 !
 route-map RM-CONN-2-BGP permit 10
    match ip address prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+   set extcommunity soo 192.168.42.1:511 additive
 !
-route-map RM-EVPN-EXPORT-VRF-DEFAULT permit 30
-   match ip address prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+route-map RM-EVPN-EXPORT-VRF-DEFAULT permit 10
+   match extcommunity soo 192.168.42.1:511 additive
```

Comment:
match additive? Also I believe this command takes the extended community list as argument.